Qt4_LimaVideo.py

```diff
@@ -42,31 +48,38 @@
 import time
 import logging
 import gevent
+import struct
 import numpy as np
+
 from PyQt4 import QtGui
-from Lima import Core
+
+try:
+    from Lima import Core
+except:
+    pass
+
+try:
+    import PyTango
```

Comment:
I would like to test this at the beamline before accepting. This hardware object become big and it would be good to keep it compact.
Maybe better is to create a TangoVideo hardware object, because when PyTango is used there is no connection to LIMA. Am I correct? If yes then it should not be under LIMA related hwobj.